program_intent_engagement/apps/api/serializers.py

```diff
@@ -1,4 +1,27 @@
-# Serializers that can be shared across multiple versions of the API
-# should be created here. As the API evolves, serializers may become more
-# specific to a particular version of the API. In this case, the serializers
-# in question should be moved to versioned sub-package.
+
+from program_intent_engagement.apps.core.models import ProgramIntent
+from rest_framework import serializers
+from program_intent_engagement.apps.core.models import ProgramIntent, CERTAINTY
+
+class ProgramIntentSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the Program Intent Model
+    """
+
+    # TODO: gotta figure out the JWT stuff for this
+    user = serializers.CharField(required=True)
+    program_uuid = serializers.UUIDField(required=True)
+    reason = serializers.CharField(required=True)
+    certainty = serializers.ChoiceField(choices = CERTAINTY, required=True)
```

Comment:
should i add a validate for certainty? [Like for exam_types under the exams serializer? ](https://github.com/edx/edx-exams/blob/f0fbecf17b76666029a502023e551e8f891cf1fe/edx_exams/apps/api/serializers.py#L36)

Reply:
we don't really need one on the way out  having just pulled it from the DB.